src/components/search/filters/FetchFullfacet.tsx

```diff
@@ -14,16 +14,27 @@ export interface Data {
 
 interface FetchDataProps {
     children: (data: Data[]) => ReactNode;
-
+    selectedFilters: string[];
 }
 
-const FetchFullfacet: React.FC<FetchDataProps> = ({ children }) => {
+const FetchFullfacet: React.FC<FetchDataProps> = React.memo(({ children, selectedFilters }) => {
     const [data, setData] = useState<Data[]>([]);
+    const [url, setUrl] = useState<string>('http://localhost:3000/api/v3/datasets/fullfacet?facets=["type","creationLocation","ownerGroup","keywords"]&fields={}');
+
+    useEffect(() => {
+        let updatedUrl = 'http://localhost:3000/api/v3/datasets/fullfacet?facets=["type","creationLocation","ownerGroup","keywords"]&fields={}';
```

Comment:
I see the url is hard coded several places. The `http://localhost:3000/api/v3`  should be in one place and can be configured, as we know for sure we need to change it later.